README.md

```diff
@@ -1,17 +1,148 @@
-# Vizro
+<p align="center">
+<img src="https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/title.svg" width="200"/>
+</p>
 
-[![Python version](https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C%203.10%20%7C%203.11-blue.svg)]()
-[![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)]()
+<div align="center" markdown="1">
 
+[![Python version](https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C%203.10%20%7C%203.11-blue.svg)](https://pypi.org/project/vizro/)
+[![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/McK-Internal/vizx-hyphen/blob/main/LICENSE.txt)
```

Comment:
Change link - this one is going to McK-Internal

Reply:
Good spot!